test.md

```diff
@@ -0,0 +1,405 @@
+---
+markmap:
+  colorFreezeLevel: 1
+  initialExpandLevel: 1
+  maxWidth: 300
+  color: '#6437FF'
+---
+
+- Atom Geometry
```

Comment:
perhaps try to do this automatically? or you already do this automatically? If later, please include the scripts, it not I believe you can do this by reading the subclass methods by traversing the builder parent class. Otherwise, it will be very hard to maintain such a long list. And may not be precise when API changes (tho we are unlikely to change in this part)

Reply:
This was actually made "semi" automatically 😅 , I still manually know the steps but I use a python markdown library to "auto-repeat"/"auto-inject" certain repetitive parts. That being said, I did have an older script that would follow the inheritance path and show which classes gave which methods. Might be worth reworking that and making this fully programmatic. Regardless, I'll definitely include the source code. 

I think this idea also falls in line with programmatically generating the docstrings versus manually cranking them out like I've done by hand.

Reply:
I think this is in good direction, it shouldn't be too hard to do the generation, let me know if you have trouble implementing it.